src/sync/sync.types.ts

```diff
@@ -0,0 +1,2 @@
+/* valid task types */
+export type TaskType = "repository" | "pull" | "commit" | "branch" | "build" | "deployment";
```

Comment:
started using this sucker all over the shop so pulled it out of installation.ts and created new types file...